packages/cf-builder-pagination/example/basic/component.js

```diff
@@ -1,6 +1,5 @@
-const React = require('react');
-const { render } = require('react-dom');
-const { PaginationBuilder } = require('../../src/index');
+import React from 'react';
```

Comment:
The actual `/lib` output should be identical. The main index.js is still commonjs module. But now we add `/es` target where ES6 modules are preserved and webpack2 or rollup can leverage that. Also `/dist` target for UMD build - I'm going to put them on cdnjs.com.